core-build/gulp-core-build/src/tasks/JestTask.ts

```diff
@@ -92,23 +103,31 @@ export class JestTask extends GulpTask<IJestConfig> {
 
     this._copySnapshots(this.buildConfig.srcFolder, this.buildConfig.libFolder);
 
-    Jest.runCLI(
-      {
-        ci: this.buildConfig.production,
-        config: configFileFullPath,
-        collectCoverageFrom: this.taskConfig.collectCoverageFrom,
-        coverage: this.taskConfig.coverage,
-        coverageReporters: this.taskConfig.coverageReporters,
-        coverageDirectory: path.join(this.buildConfig.tempFolder, 'coverage'),
-        maxWorkers: 1,
-        moduleDirectories: ['node_modules', this.buildConfig.libFolder],
-        reporters: [path.join(__dirname, 'JestReporter.js')],
-        rootDir: this.buildConfig.rootPath,
-        runInBand: true,
-        testMatch: ['**/*.test.js?(x)'],
-        testPathIgnorePatterns: this.taskConfig.testPathIgnorePatterns,
-        updateSnapshot: !this.buildConfig.production
-      },
+    // tslint:disable-next-line:no-any
+    const jestConfig: any = {
```

Comment:
>any [](start = 22, length = 3)

why can't this be typed?

Reply:
Not defined yet.

---
In reply to: [153654321](https://github.com/Microsoft/web-build-tools/pull/453#discussion_r153654321) [](ancestors = 153654321)